src/amo/components/AddonReportAbuseLink/index.js

```diff
@@ -21,55 +20,42 @@ type Props = {|
 
 type PropsFromState = {|
   abuseReport: AddonAbuseState | null,
-  loading: boolean,
 |};
 
 type InternalProps = {|
   ...Props,
   ...PropsFromState,
-  dispatch: DispatchFunc,
   i18n: I18nType,
 |};
 
-export class ReportAbuseButtonBase extends React.Component<InternalProps> {
+export class AddonReportAbuseLinkBase extends React.Component<InternalProps> {
   render(): null | React.Node {
-    const { abuseReport, addon, i18n, loading } = this.props;
+    const { abuseReport, addon, i18n } = this.props;
 
     invariant(addon, 'An add-on is required');
 
     if (abuseReport && abuseReport.message !== undefined) {
       return (
-        <div className="ReportAbuseButton ReportAbuseButton--report-sent">
-          <h3 className="ReportAbuseButton-header">
+        <div className="AddonReportAbuseLink AddonReportAbuseLink--report-sent">
+          <h3 className="AddonReportAbuseLink-header">
             {i18n.gettext('You reported this add-on')}
           </h3>
 
-          <p className="ReportAbuseButton-first-paragraph">
+          <p className="AddonReportAbuseLink-first-paragraph">
             {i18n.gettext(`We have received your report. Thanks for letting us
               know about your concerns with this add-on.`)}
           </p>
         </div>
       );
     }
 
-    const reportButtonProps: Object = {
-      to: `/feedback/addon/${addon.slug}/`,
-    };
-
     /* eslint-disable react/no-danger */
     return (
-      <div className={makeClassName('ReportAbuseButton')}>
-        <div className="ReportAbuseButton--preview">
-          <Button
-            buttonType="neutral"
-            className="ReportAbuseButton-show-more"
-            disabled={loading}
-            rel="nofollow"
-            puffy
-            {...reportButtonProps}
-          >
+      <div className={makeClassName('AddonReportAbuseLink')}>
```

Comment:
why do we need a call to makeClassname if it is a static string?

Reply:
We don't need to. It was already there, I didn't change it, just updated the string

Reply:
218c8f4b0cd4035cd53d01d0dd4e33266791f078